bundle/CMakeLists.txt

```diff
@@ -123,12 +123,24 @@ endif()
    endif()
 
   endif(CLONE_JCSDADATA)
+# rrfs-test data (fv3 and/or mpas)
+#---------------------------------
+  if(CLONE_RRFSDATA)
      set(RDAS_RRFS_DATA_ROOT "$ENV{RDAS_RRFS_DATA_ROOT}/rrfs-test-data")
-#cltorg     set(RDAS_RRFS_DATA_ROOT "/scratch2/NCEPDEV/fv3-cam/Ting.Lei/dr-emc-rdas/rrfs-test-data") #clttodo
+#    set(RDAS_RRFS_DATA_ROOT "/scratch2/NCEPDEV/fv3-cam/Ting.Lei/dr-emc-rdas/rrfs-test-data")
      set(DESTINATION_DIR "${CMAKE_CURRENT_SOURCE_DIR}")
-     file( COPY "${RDAS_RRFS_DATA_ROOT}" DESTINATION ${DESTINATION_DIR} )
+     if(NOT EXISTS "${DESTINATION_DIR}/rrfs-test-data")
```

Comment:
An problem is that there would be more rrfs (fv3 or mpas) to be added and there would no be only  those rundir . 
Maybe you could create an intermiediate dir like rrfs-fv3-data and rrfs-mpas-data

Reply:
@TingLei-NOAA so the issue is the hard coded reference to the test case: rundir-rrfs_fv3jedi_hyb_2022052619?

I could just do something like this:
```
execute_process(COMMAND find "${RDAS_RRFS_DATA_ROOT}" -type d -name "rundir-rrfs_fv3jedi*" -exec rsync -av {} "${DESTINATION_DIR}/rrfs-test-data" \;)
```
which would just rysinc anything that starts with `rundir-rrfs_fv3jedi*`. It might be better to select which you want, but that might be something we could add in the future when we have the need? For now, this seems fine to me.

Reply:
OK, we can work on this later. And now, just use your current version.